Assets/Mirror/Components/NetworkPingDisplay.cs

```diff
@@ -10,10 +10,12 @@ namespace Mirror
     [HelpURL("https://mirror-networking.com/docs/Components/NetworkPingDisplay.html")]
     public class NetworkPingDisplay : MonoBehaviour
     {
-        [SerializeField] bool showPing = true;
-        [SerializeField] Vector2 position = new Vector2(200, 0);
-        [SerializeField] int fontSize = 24;
-        [SerializeField] Color textColor = new Color32(255, 255, 255, 80);
+        public bool showPing = true;
+        public Vector2 position = new Vector2(200, 0);
+        public int fontSize = 24;
+        public Color textColor = new Color32(255, 255, 255, 80);
+        public string prefix;
```

Comment:
might be better to use this so we have 1 field instead of 2, 
```cs
string format = "{0}ms";
``` 
and then use this for the text
```cs
string text = string.format(format, (int)(NetworkTime.rtt * 1000));
```

Reply:
agreed, this is better

Reply:
i dont understand , what do you mean byt 1 field instead of 2 ?

Reply:
made the suggestions for the change